sui/src/sui_commands.rs

```diff
@@ -352,7 +354,7 @@ pub async fn genesis(
     let mut network_config = NetworkConfig {
         epoch: 0,
         authorities: vec![],
-        buffer_size: DEFAULT_MAX_DATAGRAM_SIZE,
+        buffer_size: 650000,
```

Comment:
Same const here to make sure they are all the same if they change.

Reply:
same comment as https://github.com/MystenLabs/sui/pull/1687#discussion_r862957573